### ISSUE TITLE
(Angular 16) Textarea Input Issue

### ISSUE THREAD

**Issue by wonko-gsa** [GSA/sam-design-system#1357]
**Components Involved**
`textarea`

**Expected Behavior**
I expect to see the text: “123 characters allowed” under the textarea and the width of the box to be matched as before.

**Actual Behavior**
I do not see the text: “123 characters allowed” under the textarea and the width of the box is smaller.

**Your Environment**
Angular Version: "^16.2.12"
```
    "@gsa-sam/components": "16.0.4",
    "@gsa-sam/layout": "16.0.1",
    "@gsa-sam/layouts": "16.0.0",
    "@gsa-sam/ngx-uswds": "16.0.1",
    "@gsa-sam/ngx-uswds-icons": "16.0.0",
    "@gsa-sam/sam-formly": "16.0.4",
    "@gsa-sam/sam-material-extensions": "16.0.4",
    "@gsa-sam/sam-styles": "3.0.13",
    "@gsa-sam/shared": "16.0.1",
```

**Ticket Number**
IAEMOD-21463 Angular 16 Upgrade

**Additional context**
**Issue 1.** 
a. Formly: 
```
  otherTextField: FormlyFieldConfig[] = [
    {
      key: 'otherText',
      type: 'textarea',
      templateOptions: {
        disabled: true,
        label: '',
        placeholder: 'e.g. Program Name',
        maxLength: 123,
        announceLabel: true,
        attributes: {
          style: "max-width: 200%;",
          'aria-labelledby': 'goalRadioSubOptions'
        },
      },
      hooks: {
        onChanges: (field) => {
          this.otherTextField[0].templateOptions.disabled = !this.enableTextBox();
          field.formControl.valueChanges.subscribe(value => {
            this.otherSubText = value;
          });
        }
      }
    }
  ];
```

b. A screenshot:
![image](https://github.com/GSA/sam-design-system/assets/117218338/427f7c6f-d087-46b3-982b-2bf029473d40)

**Comment by yerramshilpa** [comment]
Issue Resolved in @gsa-sam/sam-formly": "16.0.5"

**Comment by wonko-gsa** [comment]
Thank you for resolving this issue @yerramshilpa
